service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/Job.java

```diff
@@ -87,35 +56,32 @@
      * @throws IOException if the function contains invalid parameters
      * @throws IotHubException if the http request failed
      */
-    void scheduleUpdateTwin(String queryCondition,
-                            DeviceTwinDevice updateTwin,
-                            Date startTimeUtc, long maxExecutionTimeInSeconds) throws IOException, IotHubException
+    void scheduleUpdateTwin(
+        String queryCondition,
+        DeviceTwinDevice updateTwin,
+        Date startTimeUtc,
+        long maxExecutionTimeInSeconds)
+        throws IOException, IotHubException
     {
-        /* Codes_SRS_JOB_21_006: [If the updateTwin is null, the scheduleUpdateTwin shall throws IllegalArgumentException.] */
-        if(updateTwin == null)
+        if (updateTwin == null)
         {
             throw new IllegalArgumentException("null updateTwin");
```

Comment:
I know this already exists but I happened to notice the language on all these IllegalArgumentException is not a full sentence like elsewhere. Does it sound reasonable to update it?

Reply:
Sure, now is a good a time as any